apps/backoffice-v2/package.json

```diff
@@ -68,6 +68,7 @@
     "@radix-ui/react-separator": "^1.0.2",
     "@radix-ui/react-slot": "^1.0.1",
     "@tanstack/react-query": "^4.19.1",
+    "@tanstack/react-table": "^8.9.2",
```

Comment:
Using `react-table` for the `Table` cell both makes adding sorting and such trivial, and their docs has a full snippet I could copy-paste for rendering.